_posts/2022-08-26-get-active.md

```diff
@@ -0,0 +1,58 @@
+---
+author: Drew Winstel
+category: General
+date: 2022-08-26 12:00:00-05:00
+layout: post
+title: Get Active at DjangoCon US
+---
+
+🚴🏻🏃🏾🏊🏽🚶🏼 Looking to get active during DjangoCon US?
+
+# Getting active at DjangoCon US
+
+Having talks means a lot of time spent not moving while watching talks. It's important to take care of your body as well as your mind, and a good way to do that is to get moving.
+
+If you would like a little social motivation (or an excuse to share your accomplishments), we've created a [Strava group](https://www.strava.com/clubs/1073023) as we had in 2021. It's a different group from 2021, so you'll have to join again even if you joined last year.
+
+Once we send invitations to attendees to join the conference Slack instance (invitations will be sent out to ticket holders by Friday, October 14), join the `#outings` channel if you'd like to join or plan a group exercise.
+
+## At the hotel
+
+The [San Diego Marriott Mission Valley](https://www.marriott.com/en-us/hotels/sanmv-san-diego-marriott-mission-valley/experiences/) has a heated outdoor pool and hot tub as well as a fitness center.
+
+At the fitness center, you'll find:
+
+- Elliptical machines
+- Stationary bikes
+- Free weights
+- Treadmills
+
+In past years, some of this equipment has been placed outside on the covered section of the pool deck, so you can get a workout in while catching some fresh air.
+
+## Walking/running/hiking
+
+The [San Diego River Trail](https://www.sandiegoriver.org/river_trail.html) is accessible via the walkway adjacent to the Trolley [behind the hotel](/static/img/blog/access-san-diego-river-trail.png). Use the path drawn in yellow to access the trail from the hotel if you would like to take the stairs down, or use the orange section to take the ramp down to the trail from the trolley platform. It's approximately 7.1 miles (11.4 km) from the hotel to the end of the trail at the beach if you feel like catching a little coastline action.
+
+<img src="/static/img/blog/tecolote-canyon-exit.jpg" alt="Photo of an uphill gravel trail at Tecolote Canyon Natural Park" title="One of the many uphill exits in Tecolote Canyon" style="width:30%; display:block; float:right;" />
+
+If you want something unpaved, [Tecolote Canyon Natural Park & Nature Center](https://www.sandiego.gov/park-and-recreation/parks/osp/tecolote) is about 6 miles from the hotel (10-15 minute drive) and has some stunning scenery with lots of exits (mostly uphill on the way out because it's a canyon) that you can bail out at. Make sure you bring a _lot_ of water. It's great for taking a taxi (or Uber/Lyft) to the nature center on your way in, then hiking for as long as you like, then catching a ride back once you're done without having to hike all the way back.
+
+[Mission Trails Regional Park](https://mtrp.org/trail-maps/) has 65 miles of trails, the majority of which are open to hikers and mountain bikers. Some of the trails are quite steep, so know your limits!
```

Comment:
Two more walking areas that are fun: 1) Balboa Park, which is pretty close and fairly flat and 2) Torrey Pines State Park which has free entry but paid parking... it's further away but it has some great views due to being right above the beach

Reply:
Added. Thanks!